src/App/App.scss

```diff
@@ -1 +1,6 @@
 @import "../variables";
+
```

Comment:
We had an issue where you could scroll horizontally a bit on mobile. This fixes it.